# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xulrunner/hotfix-startup-notification-includepath.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mozilla/widget/src/gtk2/nsWindow.cpp.orig	2008-05-06 06:11:52.000000000 +0200
+++ mozilla/widget/src/gtk2/nsWindow.cpp	2008-08-26 15:12:43.000000000 +0200
@@ -65,7 +65,7 @@
 
 #ifdef MOZ_ENABLE_STARTUP_NOTIFICATION
 #define SN_API_NOT_YET_FROZEN
-#include <startup-notification-1.0/libsn/sn.h>
+#include <libsn/sn.h>
 #endif
 
 #include "gtk2xtbin.h"
